nrm_django/nrm_app/admin.py

```diff
@@ -1,3 +1,23 @@
-# from django.contrib import admin
+from django.contrib import admin
 
 # Register your models here.
+
+
+class NRMAdmin(admin.ModelAdmin):
+
+    """Extra context added to the default ModelAdmin."""
+
+    def get_search_description(self):
+        try:
+            desc = self.search_description
+        except AttributeError:
+            return ""
+        return desc
+
+    def changelist_view(self, request, extra_context=None):
+        extra_context = extra_context or {}
+        extra_context["search_description"] = self.get_search_description()
```

Comment:
This will definitely work. Would it be simpler, though, to use {% firstof search_description 'Default text here' %} in the search_form, then, where you want to, pass search_description into context either from the URL or view? Not sure, just wondering.

Reply:
I think that the reason it didn't work when I first tried that is that `search_form` is a template tag that doesn't have access to the context, so it had to be plumbed all the way in explicitly.